contrib/systemd/master/atomic-openshift-master-controllers

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
+
+set -euo pipefail
+
+pods_dir=/etc/origin/node/pods
+pods_disabled_dir=/etc/origin/node/pods.disabled
+
+static_pod_file=controller.yaml
+
+# Make sure both directories exist
+mkdir -p $pods_dir $pods_disabled_dir
+
+function get_container_values() {
+    child_container=$(docker ps -q --filter "label=io.kubernetes.container.name=controllers")
+    container=$(docker ps -q --filter "label=openshift.io/component=controllers" --filter "label=io.kubernetes.container.name=POD")
+}
+
+
+# See how we were called.
+case "$1" in
+start)
+    # See if controllers pod is started
+    if [ -f "${pods_dir}/${static_pod_file}" ]; then
+        echo "controllers already started"
+        exit 0
+    fi
+
+    # Start the controllers pod
+    mv "${pods_disabled_dir}/${static_pod_file}" "${pods_dir}"
+
+    sleep 1 # avoid race condition
+
+    # Wait for the pod to come up
+    while ! docker ps -q --filter "label=io.kubernetes.container.name=controllers" &> /dev/null ; do
+        sleep 1
+    done
+
+    get_container_values
+    docker logs --follow "$child_container"
+    exit $?
+    ;;
+
+stop)
+    # See if controllers pod is started
+    if [ -f "${pods_disabled_dir}/${static_pod_file}" ]; then
+        echo "controllers already stopped"
+        exit 0
+    fi
+   
+    get_container_values
+
+    # Start the controllers pod
+    mv "${pods_dir}/${static_pod_file}" "${pods_disabled_dir}"
+
+    exec timeout 60 docker wait $child_container
```

Comment:
same thing here, the timeout durations with systemd and the script should probably be synced